robotino_controller_configuration_gazebo/launch/hand_controller.launch

```diff
@@ -9,7 +9,11 @@
       <node name="hand_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="joint_group_position_controller"/>
 
       <node name="joint_states_relay_hand" pkg="topic_tools" type="relay" args="joint_states /joint_states" cwd="node" respawn="true" output="screen"/>  
-
+      
     </group> 
 
+      <node name="hand_joint_state_pub_sim" pkg="kclhand_control" type="hand_joint_state_pub_sim.py" cwd="node" respawn="true">
```

Comment:
@lokalmatador with this line the hand control will be loaded, the argument respawn=true , will spawn again the node if it crash